source/usage-examples/insertMany.txt

```diff
@@ -0,0 +1,54 @@
+=========================
+Insert Multiple Documents
+=========================
+
+.. default-domain:: mongodb
+
+You can insert a multiple documents into a collection in a single
+operation by calling the ``insertMany()`` method on a ``MongoCollection``
+object. To insert them, add your ``Document`` objects that contain the
+fields and values that you want to store to a ``List`` and pass it to
+your call to ``insertMany()``. If you call the ``insertMany()`` method on
```

Comment:
The second sentence of this paragraph is a little hard to parse -- can we simplify it at all? I think the "objects that contain the fields and values that you want to store" phrase is a little too wordy. Can we just say "objects" instead? I think we should also change "to a ``List``" into "into a ``List``".

Reply:
I'll simplify by omitting "fields and values that you want to store" -- I added this to make it more specific, but I guess it could be too verbose.

I'm not sure whether there's anything incorrect about "add to", but "add into" is an idiom which we should avoid.